portfolio/src/main/webapp/script.js

```diff
@@ -21,8 +21,36 @@ function addRandomGreeting() {
   // Picks a random quote and adds it to the page
 }
 
-function getData() {
-  fetch('/data').then(response => response.text()).then((msg) => {
-    document.getElementById('quote-container').innerText = msg;
+function getValues() {
+  fetch('/data').then(response => response.json()).then((msg) => {
+      var messages = String(msg);
```

Comment:
Nit: I believe the recommendation is now to use "let" instead of "var": https://www.w3schools.com/js/js_let.asp